components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2389,13 +2389,17 @@ where
                 .tablet_factory
                 .tablet_path(new_region.get_id(), RAFT_INIT_LOG_INDEX);
             to_cloned.push(tablet_path);
+            let mut output_stream = CodedOutputStream::vec(&mut apply_result);
```

Comment:
How about sharing the output_stream outside the loop and only flush once?

Reply:
This is memory operation. Not sure if it helps in terms of performance. 
And each flush only does the new data, so the total size is same.
What's more, internally it will flush anyway when exceeding a threshold. I think make it local is more clear.